packages/flutter/test/material/segmented_button_test.dart

```diff
@@ -1122,6 +1122,52 @@ void main() {
         ),
     );
   }, skip: kIsWeb && !isSkiaWeb); // https://github.com/flutter/flutter/issues/99933
+
+  testWidgets('SegmentedButton vertical aligned children', (WidgetTester tester) async {
```

Comment:
Can you add a golden file test as well? Look for `matchesGoldenFile` in the test directory for an example.

Reply:
I added a golden test , but i add it as a separated test , should i make it within the same old test scoop "SegmentedButton vertical aligned children" ?

Reply:
"Can you take a look at the analysis failures?"
I see "Linux analyze" some times fail , but i don't think it has something with my PR, its randomly keep fail and success , point me if i am wrong :)
+ Am sorry, i should update with rebase but i always forget.